tensorrt_llm/_torch/memory_buffers_util.py

```
@@ -0,0 +1,63 @@
+                return runtime_buffer
+            if runtime_buffer is None:
+                return pinned_buffer
+            return runtime_buffer if runtime_buffer[0].numel(
+            ) > pinned_buffer[0].numel() else pinned_buffer
+
+        def view_to(buffer: torch.Tensor, dtype: torch.dtype,
+                    tensor_shape: list[int]) -> torch.Tensor:
+            return buffer[0:math.prod(tensor_shape) *
+                          dtype.itemsize].view(dtype).view(tensor_shape)
+
+        def get_empty(tensor_shape: list[int], dtype: torch.dtype,
```

Ruff (0.12.2)

Undefined name `torch`

(F821)